actors/miner/src/lib.rs

```diff
@@ -4622,3 +4622,280 @@ impl ActorCode for Actor {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Please move this to a different testing file and directory

Reply:
I now realize that you probably can't move this to a different directory, a sibling file miner_internal_test.rs should be fine.  Also nit: `mod internal_tests`